storage/spanstore/downsampling_writer_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
2019, did you copy paste this? It shouldn't be necessary, the make fmt step should add the correct license for you

Reply:
ic, i will remove those license then.